src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -29,10 +29,13 @@ public TaskDto getTask(@RequestParam Long taskId) throws TaskNotFoundException {
     }
 
     @RequestMapping(method = RequestMethod.DELETE, value = "deleteTask")
-    public void deleteTask(Long taskId){
+    public void deleteTask(@RequestParam Long taskId)throws TaskNotFoundException{
+        System.out.println("Delete: " + "id: "+getTask(taskId).getId() + " title: " + getTask(taskId).getTitle()
+                + " content: " + getTask(taskId).getContent());
+        service.deleteTask(taskId);
     }
 
-    @RequestMapping(method = RequestMethod.PUT, value = "updateTask")
+    @RequestMapping(method = RequestMethod.PUT, value = "updateTask", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
@RestController domyslnie konsumuje application_json_value, wiec to nie jest potrzebne